benchmarks/benchmarks/generate_data/um_files.py

```diff
@@ -22,12 +22,21 @@ def _create_um_files(
     from tempfile import NamedTemporaryFile
 
     from mule import ArrayDataProvider, Field3, FieldsFile
+    import mule.ff
     from mule.pp import fields_to_pp_file
     import numpy as np
 
     from iris import load_cube
     from iris import save as save_cube
 
+    def to_bytes_patch(self, field):
+        data = field.get_data()
+        dtype = mule.ff._DATA_DTYPES[self.WORD_SIZE][field.lbuser1]
+        data = data.astype(dtype)
+        return data.tobytes(), data.size
+
+    mule.ff._WriteFFOperatorUnpacked.to_bytes = to_bytes_patch
```

Comment:
Please could we have a `# TODO:` comment explaining why this is here, and that it's temporary? Could maybe cross reference that discussion I posted on `simulation-systems` (it's publicly visible).